nilearn/signal.py

```diff
@@ -810,16 +812,18 @@ def clean(
 
 
 def _handle_scrubbed_volumes(
-    signals, confounds, sample_mask, filter_type, t_r
+    signals, confounds, sample_mask, filter_type, t_r, extrapolate=False
```

Comment:
```suggestion
    signals, confounds, sample_mask, filter_type, t_r, extrapolate
```
Private function should not have default parameter. I don't remember the reason other then my style checker yelled at me before. @Remi-Gau is this is something we should impose?

Reply:
> Private function should not have default parameter. 

Your style checker is complaining about this? Does it point to a "reference" explaining why? That's not a recommendation I am familiar with but then again my python coding style knowledge is that of a trained baboon so...

Also don't show some part of the nilearn code base to your style checker otherwise your screen will light up like its Xmas time because we do have several private functions with default values.

Reply:
Taking this as a commonsense comment: I think that default parameters should not be burried deep in the codebase, but visible in the API so that the behavior is easy to figure out. My 2c,

Reply:
ha yes in that sense I agree: though I am sure we can think of cases where you may not want to have all the parameters of all the private functions exposed in the public API.
I think I was more surprised by a style checker seemingly making some absolute (ish) recommendation on this.

Reply:
Again, the reasoning for doing so was to preserve compatibility with potential usages of the function (not inside the package perhaps, but outside). I am ok with adding the `extrapolate` parameter without a predefined value, specially if we want to make it available to the top-level function `clean()`.

Reply:
No I cannot find PEP8 reference for this. I can see we agree that behaviour changing parameter should be exposed.